src/doc/en/faq/faq-general.rst

```diff
@@ -334,8 +334,10 @@ Here is a BibTeX entry for Sage:
     }
 
 Adjust version/year as needed.  You might also like to use DOI for Sage,
-as the note entry in the above record, or directly as DOI record.
-
+as the ``note`` entry in the above record, or directly as DOI record.
+The DOI :doi:`10.5281/zenodo.8042260` represents all versions of Sage;
+clicking on it will show the DOI for the latest Sage version
+(at the time of writing, 10.3, see :doi:`10.5281/zenodo.10841614`).
 
```

Comment:
Do you really want to keep this part 
```
(at the time of writing, 10.3, see :doi:`10.5281/zenodo.10841614`)
```
(which seems redundant to me)?

Reply:
as I said, it's for illustrative purposes - cause you click on one DOI, but get another DOI...

Reply:
OK.

Reply:
Do you want to update the example BibTeX entry or leave it as it is now?

Reply:
the example bibtex entry, normally speaking, ought to refer to a concrete version of Sage, not "the latest available"

Reply:
OK. No problem.